templates/cert-manager/00-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.certManager.create }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.certManager.namespace }}
+{{- end }}
```

Comment:
I think was just missed when things were broken into separate files: https://github.com/revelrylabs/moondog-engine/pull/72/files#diff-92fcfd651493f56de0ddbf9b87e7a2d9R3-R6

Reply:
weird that link just pulls up the same page for me. but I believe you.

Reply:
Yeah that link was broken, my bad. Here's the NS in the main `cert-manager.yaml` file:
https://github.com/revelrylabs/moondog-engine/blob/1b70e8eab7ce47c5783115cf53a8ad02ae0a4742/templates/cert-manager/cert-manager.yaml#L1-L6